python/lsst/ts/mtdomecom/mtdome_com.py

```diff
@@ -675,23 +786,77 @@ async def set_temperature(self, temperature: float) -> None:
             command=CommandName.SET_TEMPERATURE, temperature=temperature
         )
 
-    async def exit_fault(self) -> None:
-        """Indicate that all hardware errors have been resolved."""
-        # For backward compatibility with XML 12.0, we always send resetDrives
-        # commands.
+    async def exit_fault(self, sub_system_ids: int) -> None:
+        """Indicate that all hardware errors have been resolved.
+
+        Parameters
+        ----------
+        sub_system_ids : `int`
+            Bitmask of the sub-systems to set the operational mode for.
+
+        """
+        self.log.debug(f"WOUTER exit_fault: {sub_system_ids=!s}")
```

Comment:
WOUTER --> Are you sure to put your name here?

Reply:
Removed.